### ISSUE TITLE
2021-05

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#14]

**Comment by qiaojy19** [comment]
# May - 2021

## 05.31 - 06.04
| Title                                                                                                                                           |
| ----------------------------------------------------------------------------------------------------------------------------------------------- |
| [Federated Learning with Fair Averaging](https://arxiv.org/abs/2104.14937)                                                                      |
| [Blockchain based Privacy-Preserved Federated Learning for Medical Images: A Case Study of COVID-19 CT Scans](https://arxiv.org/abs/2104.10903) |
| [DRIVE: One-bit Distributed Mean Estimation](https://arxiv.org/abs/2105.08339)                                                                  |
| [Dynamic Scheduling for Over-the-Air Federated Edge Learning with Energy Constraints](https://arxiv.org/abs/2106.00490)                         |
| [Early Detection of COVID-19 Hotspots Using Spatio-Temporal Data](https://arxiv.org/abs/2106.00072)                                             |
| [Energy-Efficient and Federated Meta-Learning via Projected Stochastic Gradient Ascent](https://arxiv.org/abs/2105.14772)                       |
| [FedBABU: Towards Enhanced Representation for Federated Image Classification](https://arxiv.org/abs/2106.06042)                                 |
| [FedCCEA : A Practical Approach of Client Contribution Evaluation for Federated Learning](https://arxiv.org/abs/2106.02310)                     |
| [Federated Estimation of Causal Effects from Observational Data](https://arxiv.org/abs/2106.00456)                                              |
| [Federated Neural Collaborative Filtering](https://arxiv.org/abs/2106.04405)                                                                    |
| [FedHealth 2: Weighted Federated Transfer Learning via Batch Normalization for Personalized Healthcare](https://arxiv.org/abs/2106.01009)       |
| [FedNL: Making Newton-Type Methods Applicable to Federated Learning](https://arxiv.org/abs/2106.02969)                                          |
| [H-FL: A Hierarchical Communication-Efficient and Privacy-Protected Architecture for Federated](https://arxiv.org/abs/2106.00275)               |
| [Local Adaptivity in Federated Learning: Convergence and Consistency](https://arxiv.org/abs/2106.02305)                                         |
| [Membership Inference Attacks on Deep Regression Models for Neuroimaging](https://arxiv.org/abs/2105.02866)                                     |
| [Meta-HAR: Federated Representation Learning for Human Activity Recognition](https://arxiv.org/abs/2106.00615)                                  |
| [On Dynamic Resource Allocation for Blockchain Assisted Federated Learning over Wireless Channels](https://arxiv.org/abs/2105.14708)            |
| [Preservation of the Global Knowledge by Not-True Self Knowledge Distillation in Federated Learning](https://arxiv.org/abs/2106.03097)          |
| [QLSD: Quantised Langevin stochastic dynamics for Bayesian federated learning](https://arxiv.org/abs/2106.00797)                                |
| [SemiFL: Communication Efficient Semi-Supervised Federated Learning with Unlabeled Clients](https://arxiv.org/abs/2106.01432)                   |
| [SpreadGNN: Serverless Multi-task Federated Learning for Graph Neural Networks](https://arxiv.org/abs/2106.02743)                               |
| [Triggerflow: Trigger-based Orchestration of Serverless Workflows](https://arxiv.org/abs/2106.00583)                                            |
| [Unifying Distillation with Personalization in Federated Learning](https://arxiv.org/abs/2105.15191)                                            |
| [Wireless Federated Learning with Limited Communication and Differential Privacy](https://arxiv.org/abs/2106.00564)                             |
## 05.24 - 05.28
| Title                                                                                                                                |
| ------------------------------------------------------------------------------------------------------------------------------------ |
| [A Federated Learning Framework for Nonconvex-PL Minimax Problems](https://arxiv.org/abs/2105.14216)                                 |
| [Accelerated Gradient Tracking over Time-varying Graphs for Decentralized Optimization](https://arxiv.org/abs/2104.02596)            |
| [Concept drift detection and adaptation for federated and continual learning](https://arxiv.org/abs/2105.13309)                      |
| [Designing ECG Monitoring Healthcare System with Federated Transfer Learning and Explainable AI](https://arxiv.org/abs/2105.12497)   |
| [Federated Graph Learning -- A Position Paper](https://arxiv.org/abs/2105.11099)                                                     |
| [Federated Learning for Industrial Internet of Things in Future Industries](https://arxiv.org/abs/2105.14659)                        |
| [Federated Learning for Short-term Residential Energy Demand Forecasting](https://arxiv.org/abs/2105.13325)                          |
| [Fed-NILM: A Federated Learning-based Non-Intrusive Load Monitoring Method for Privacy-Protection](https://arxiv.org/abs/2105.11085) |
| [FedScale: Benchmarking Model and System Performance of Federated Learning](https://arxiv.org/abs/2105.11367)                        |
| [Networked Federated Multi-Task Learning](https://arxiv.org/abs/2105.12769)                                                          |
| [PPT: A Privacy-Preserving Global Model Training Protocol for Federated Learning in P2P Networks](https://arxiv.org/abs/2105.14408)  |
| [Quantum Federated Learning with Quantum Data](https://arxiv.org/abs/2106.00005)                                                     |
| [Towards a Federated Learning Framework for Heterogeneous Devices of Internet of Things](https://arxiv.org/abs/2105.14675)           |

## 05.17 - 05.21
| Title                                                                                                                                                       |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [DID-eFed: Facilitating Federated Learning as a Service with Decentralized Identities](https://arxiv.org/abs/2105.08671)                                    |
| [Fast Federated Learning by Balancing Communication Trade-Offs](https://arxiv.org/abs/2105.11028)                                                           |
| [Byzantine-Resilient Federated Machine Learning via Over-the-Air Computation](https://arxiv.org/abs/2105.10883)                                             |
| [Federated Learning with Fair Averaging](https://arxiv.org/abs/2104.14937)                                                                                  |
| [A Dispersed Federated Learning Framework for 6G-Enabled Autonomous Driving Cars](https://arxiv.org/abs/2105.09641)                                         |
| [A Privacy-Preserving Approach to Extraction of Personal Information through Automatic Annotation and Federated Learning](https://arxiv.org/abs/2105.09198) |
| [Adaptive ABAC Policy Learning: A Reinforcement Learning Approach](https://arxiv.org/abs/2105.08587)                                                        |
| [Business analytics meets artificial intelligence: Assessing the demand effects of discounts on Swiss train tickets](https://arxiv.org/abs/2105.01426)      |
| [Clustered Sampling: Low-Variance and Improved Representativity for Clients Selection in Federated Learning](https://arxiv.org/abs/2105.05883)              |
| [Data-Free Knowledge Distillation for Heterogeneous Federated Learning](https://arxiv.org/abs/2105.10056)                                                   |
| [Denoising Noisy Neural Networks: A Bayesian Approach with Compensation](https://arxiv.org/abs/2105.10699)                                                  |
| [DRIVE: One-bit Distributed Mean Estimation](https://arxiv.org/abs/2105.08339)                                                                              |
| [EasyFL: A Low-code Federated Learning Platform For Dummies](https://arxiv.org/abs/2105.07603)                                                              |
| [Fed-EINI: An Efficient and Interpretable Inference Framework for Decision Tree Ensembles in Federated Learning](https://arxiv.org/abs/2105.09540)          |
| [Federated Knowledge Graphs Embedding](https://arxiv.org/abs/2105.07615)                                                                                    |
| [Federated Learning With Highly Imbalanced Audio Data](https://arxiv.org/abs/2105.08550)                                                                    |
| [Federated Singular Vector Decomposition](https://arxiv.org/abs/2105.08925)                                                                                 |
| [HyFed: A Hybrid Federated Framework for Privacy-preserving Machine Learning](https://arxiv.org/abs/2105.10545)                                             |
| [Private Hierarchical Clustering in Federated Networks](https://arxiv.org/abs/2105.09057)                                                                   |
| [Prototype Guided Federated Learning of Visual Feature Representations](https://arxiv.org/abs/2105.08982)                                                   |
| [Semi-Decentralized Federated Edge Learning for Fast Convergence on Non-IID Data](https://arxiv.org/abs/2104.12678)                                         |
| [Separation of Powers in Federated Learning](https://arxiv.org/abs/2105.09400)                                                                              |
| [Towards Unsupervised Domain Adaptation for Deep Face Recognition under Privacy Constraints via Federated Learning](https://arxiv.org/abs/2105.07606)       |
| [User Label Leakage from Gradients in Federated Learning](https://arxiv.org/abs/2105.09369)                                                                 |
## 05.10 - 05.14
| Title                                                                                                                                          |
| ---------------------------------------------------------------------------------------------------------------------------------------------- |
| [DP-SIGNSGD: When Efficiency Meets Privacy and Robustness](https://arxiv.org/abs/2105.04808)                                                   |
| [FL-NTK: A Neural Tangent Kernel-based Framework for Federated Learning Convergence Analysis](https://arxiv.org/abs/2105.05001)                |
| [Slashing Communication Traffic in Federated Learning by Transmitting Clustered Model Updates](https://arxiv.org/abs/2105.04153)               |
| [An Efficient Learning Framework For Federated XGBoost Using Secret Sharing And Distributed Optimization](https://arxiv.org/abs/2105.05717)    |
| [Clustered Sampling: Low-Variance and Improved Representativity for Clients Selection in Federated Learning](https://arxiv.org/abs/2105.05883) |
| [Federated Learning with Unreliable Clients: Performance Analysis and Mechanism Design](https://arxiv.org/abs/2105.06256)                      |
| [Federated Unbiased Learning to Rank](https://arxiv.org/abs/2105.04761)                                                                        |
| [From Distributed Machine Learning to Federated Learning: A Survey](https://arxiv.org/abs/2104.14362)                                          |
| [Node Selection Toward Faster Convergence for Federated Learning on Non-IID Data](https://arxiv.org/abs/2105.07066)                            |
| [OpenFL: An open-source framework for Federated Learning](https://arxiv.org/abs/2105.06413)                                                    |
| [Privacy-Preserving Constrained Domain Generalization for Medical Image Classification](https://arxiv.org/abs/2105.08511)                      |
| [Privacy-preserving Logistic Regression with Secret Sharing](https://arxiv.org/abs/2105.06869)                                                 |
| [Separate but Together: Unsupervised Federated Learning for Speech Enhancement from Non-IID Data](https://arxiv.org/abs/2105.04727)            |
| [Stronger Privacy for Federated Collaborative Filtering with Implicit Feedback](https://arxiv.org/abs/2105.03941)                              |
| [The FeatureCloud AI Store for Federated Learning in Biomedicine and Beyond](https://arxiv.org/abs/2105.05734)                                 |
| [The Federated Tumor Segmentation (FeTS) Challenge](https://arxiv.org/abs/2105.05874)                                                          |
| [Towards Practical Watermark for Deep Neural Networks in Federated Learning](https://arxiv.org/abs/2105.03167)                                 |
## 05.03 - 05.07
| Title                                                                                                                                                   |
| ------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Byzantine-Robust and Privacy-Preserving Framework for FedML](https://arxiv.org/abs/2105.02295)                                                         |
| [Membership Inference Attacks on Deep Regression Models for Neuroimaging](https://arxiv.org/abs/2105.02866)                                             |
| [A Family of Hybrid Federated and Centralized Learning Architectures in Machine Learning](https://arxiv.org/abs/2105.03288)                             |
| [Accelerated Gradient Tracking over Time-varying Graphs for Decentralized Optimization](https://arxiv.org/abs/2104.02596)                               |
| [Business analytics meets artificial intelligence: Assessing the demand effects of discounts on Swiss train tickets](https://arxiv.org/abs/2105.01426)  |
| [Density-Aware Federated Imitation Learning for Connected and Automated Vehicles with Unsignalized Intersection](https://arxiv.org/abs/2105.01889)      |
| [Federated Face Recognition](https://arxiv.org/abs/2105.02501)                                                                                          |
| [Federated Multi-View Learning for Private Medical Data Integration and Analysis](https://arxiv.org/abs/2105.01603)                                     |
| [FedGL: Federated Graph Learning Framework with Global Self-Supervision](https://arxiv.org/abs/2105.03170)                                              |
| [Latency Analysis of Consortium Blockchained Federated Learning](https://arxiv.org/abs/2105.04087)                                                      |
| [Loss Tolerant Federated Learning](https://arxiv.org/abs/2105.03591)                                                                                    |
| [Personalized Federated Learning by Structured and Unstructured Pruning under Data Heterogeneity](https://arxiv.org/abs/2105.00562)                     |
| [Stronger Privacy for Federated Collaborative Filtering with Implicit Feedback](https://arxiv.org/abs/2105.03941)                                       |
| [The Tags Are Alright: Robust Large-Scale RFID Clone Detection Through Federated Data-Augmented Radio Fingerprinting](https://arxiv.org/abs/2105.03671) |
| [Towards Practical Watermark for Deep Neural Networks in Federated Learning](https://arxiv.org/abs/2105.03167)                                          |
